### ISSUE TITLE
Title "Drawer" - only effecting the clips in drawer.

### ISSUE THREAD

**Issue by belisoful** [CommandPost/FCPCafe#387]
Apple Feedback Assistant ID: MISSING!
---

**Is your feature request related to a problem? Please describe.**
I'd like some Titles with effects to only manipulate the pixels of a certain layer or set of layers rather than just _all_ the layers below.
This would be a really nice option to limit the scope of a Title layer.

**Describe the solution you'd like**
On a Title effect bar or within the "video animation" panel header, a circle with a "+" within it.
Upon clicking, a "drawer" opens up under the Title Effect.  A small space is made for the drawer under the Title Effect.
The Drawer has a "hamburger" style click control.  Upon clicking, the "drawer" can be opened to include layers below it to the lowest most layer in the project.

This allows only one or a couple of layers to be effected by the Title manipulation, but not all.

Clicking the activated "(+)" will retract the drawer and effect all layers as normal.

The drawer can be opened to any level with or without a clip. maybe "snapping to minimize" to the highest lowest included level.   It is the option to collapse the drawer to the smallest size without effecting which clips are included..

**Describe alternatives you've considered**
Clips can be used to accumulate layers to only affect those particular elements....   but this solution is elegant, and Apple-like.

**Additional context**
Title effects Clip A, B, & C
________________
My Title.           (+)|
^^^^^^^^^^^^^^
_____________
Clip A.             |
^^^^^^^^^^^^
____________________
Clip B.                           |
^^^^^^^^^^^^^^^^^
--------------------------- <- Magnetic Timeline
_____________
Clip C.             |
^^^^^^^^^^^^
---------------------------

clicking "(+)"
________________
My Title.           (*)|
^^^^^^^^^^^^^^
===--------------|.  <- Drawer
_____________    
Clip A.             |    
^^^^^^^^^^^^  
....

and dragging to include other clips

________________
My Title.           (*)|
^^^^^^^^^^^^^^
_____________    |
Clip A.             |    |
^^^^^^^^^^^^  |
_____________    |
_______________|____
Clip B.                  |        |
^^^^^^^^^^^^^|^^^
===--------------     <- Drawer,  Title only affects Clip A & B for specified time.

---------------------------
_____________
Clip C.             |
^^^^^^^^^^^^
---------------------------

**Comment by latenitefilms** [comment]
I assume you're aware of the "Behind" blend mode?

**Comment by belisoful** [comment]
Yes.  This is about only applying a "title" to a specific number of clips below it rather than just applying the title to all clips below it.

In Motion, the SOURCE wouldn't be all the clips below the title but only the clips in the drawer.  The default, of course, is that the drawer includes all clips below it.  That is the expected typical behavior.

I think this would be really useful as a video technician.  My main experience is actually in software development, and I'm just applying software "abstraction" methods to video editing.  fun.
